src/oop/oop2.py

```diff
@@ -4,19 +4,44 @@
 # object is constructed.
 
 class GroundVehicle():
-    def __init__(self, num_wheels):
+    def __init__(self, num_wheels = 4):
         self.num_wheels = num_wheels
 
+    def __repr__(self):
+        print(f"This vehicle has " + str(self.num_wheels) +  " wheels.")   
+
+    def drive(self):
+        return "vroooom"    
+
     # TODO
 
 
+## When I print these I get a third print in the middle of them that says None, what does that refer to?
+gv = GroundVehicle(6)
+print(gv.__repr__())
+print(gv.drive())
+
+
+
 # Subclass Motorcycle from GroundVehicle.
 #
 # Make it so when you instantiate a Motorcycle, it automatically sets the number
 # of wheels to 2 by passing that to the constructor of its superclass.
 #
 # Override the drive() method in Motorcycle so that it returns "BRAAAP!!"
 
+class Motorcycle(GroundVehicle):
+    def __init__(self, num_wheels = 4):
+        GroundVehicle.__init__(self, num_wheels = 2)
```

Comment:
You could do it this way or you could use super and do 
```suggestion
        def __init__(self, num_wheels = 2):
            super().__init__(num_wheels)
```